state-sync/diem-data-client/src/diemnet/mod.rs

```diff
@@ -434,17 +441,38 @@ impl DataSummaryPoller {
             let storage_summary = match result {
                 Ok(storage_summary) => storage_summary,
                 Err(error) => {
-                    trace!(
-                        "Error encountered when polling peer ({:?})! Error: {:?}",
-                        peer,
-                        error
+                    debug!(
+                        (LogSchema::new(LogEntry::StorageSummaryResponse)
+                            .event(LogEvent::PeerPollingError)
+                            .message(&format!(
+                                "Error encountered when polling peer {:?}. Error: {:?}",
+                                peer, error
+                            )))
                     );
                     continue;
                 }
             };
 
+            info!(
+                (LogSchema::new(LogEntry::StorageSummaryResponse)
+                    .event(LogEvent::PeerPollingSuccess)
+                    .message(&format!(
+                        "Successfully polled peer {:?}. Storage summary: {:?}",
+                        peer, storage_summary
+                    )))
+            );
+
             self.data_client.update_summary(peer, storage_summary);
             self.data_client.update_global_summary_cache();
+
+            info!(
+                (LogSchema::new(LogEntry::PeerStates)
+                    .event(LogEvent::AggregateSummary)
+                    .message(&format!(
+                        "Global data summary: {:?}",
+                        self.data_client.get_global_data_summary()
+                    )))
+            );
```

Comment:
Unless this information would be used to determine an action, I'd keep anything that occurs on a regular basis to a `debug` level unless it's a simple message without too much variance in it.